ext/js/lib/hash.rb

```diff
@@ -0,0 +1,8 @@
+require "json"
+
+class Hash
+  # Convert a hash to a JavaScript object
+  def to_js
+    JS.eval("return #{self.to_json}")
```

Comment:
Could you avoid using `to_json` to implement `Hash#to_js`? If I understand correctly, this doesn't handle non-POD objects. e.g. Consider what happens when calling `to_js` against `{ key: JS.global }`

Therefore I suggest using the same way as `Array#to_js` like:

```ruby
class Hash
  def to_js
    new_object = JS.eval("return {}")
    self.each do |key, value|
      new_object[key] = value
  end
  new_object
end
```